test/test_transformers.py

```diff
@@ -722,6 +723,43 @@ def set_weights_deterministic(model):
         self.assertEqual(result.shape, ref_output.shape)
         torch.testing.assert_close(result, ref_output, atol=1e-3, rtol=1e-2)
 
+    @parametrize("type", ["dense", "nested"])
```

Comment:
@cpuhrsch bare minimum test added here but since we don't have any ci/cd setup to run this code. How do you think I should expand on these. Obviously wan't to add tests that check the dispatch mechanism